pkg/server/server.go

```diff
@@ -407,11 +426,24 @@ func (s *Server) StreamedListObjects(req *openfgav1.StreamedListObjectsRequest,
 	)
 
 	req.AuthorizationModelId = typesys.GetAuthorizationModelID() // the resolved model id
-	return q.ExecuteStreamed(
+
+	resolutionMetadata, err := q.ExecuteStreamed(
 		typesystem.ContextWithTypesystem(ctx, typesys),
 		req,
 		srv,
 	)
+	if err == nil {
```

Comment:
Same comment as in https://github.com/openfga/openfga/pull/959/files#r1310778211